src/scalper.ts

```diff
@@ -702,14 +702,14 @@ Spot Δ: ${spotDelta} Offset Δ ${this.deltaOffset} Fair Value: ${fairValue}`,
     }
 
     // Send the delta hedge order to openbook.
-    console.log(this.symbol, 'Sufficient liquidity. Sweep OpenBook');
+    console.log(this.symbol, 'Sweep OpenBook');
     const amountDelta = roundQtyToSpotSize(Math.abs(hedgeDeltaClip), this.minSpotSize);
     const priceDelta = roundPriceToTickSize(Math.abs(hedgePrice), this.tickSize);
     const payerAccount = getPayerAccount(hedgeSide, this.symbol, 'USDC');
     console.log(this.symbol, hedgeSide, 'OpenBook-SPOT', amountDelta, 'Limit:', priceDelta, '#', deltaHedgeCount, 'ID', deltaID.toString());
     const deltaOrderTx = new Transaction();
     const deltaTx = await spotMarket.makePlaceOrderTransaction(this.connection, {
-      owner: new Account(this.owner.secretKey),
+      owner: this.owner.publicKey,
```

Comment:
whats the reason for this? was it broken before?

Reply:
Copied over from when we had used placeOrder() on the delta orders. Now it matches the usage with the gamma orders

Reply:
Both work fine. I had been debugging here for ahwile though since our OO account on OPB need to be init before the makePlace worked